app/Config/core.php

```diff
@@ -113,25 +113,25 @@
  * will override the automatic detection of full base URL and can be
  * useful when generating links from the CLI (e.g. sending emails)
  */
-	//Configure::write('App.fullBaseUrl', 'http://example.com');
+    //Configure::write('App.fullBaseUrl', 'http://example.com');
```

Comment:
__[staging][code_sniffer]__
* No space found before comment text; expected "// Configure::write('App.fullBaseUrl', 'http://example.com');" but found "//Configure::write('App.fullBaseUrl', 'http://example.com');"
* Inline comments must end in full-stops, exclamation marks, or question marks
* There must be no blank line following an inline comment